streaming_jupyter_integrations/magics.py

```diff
@@ -405,7 +405,9 @@ async def display_execution_result(self, execution_result: TableResult, display_
                 display_handle = None
                 for result in results:
                     # Explicit await for the same reason as in `__internal_execute_sql`
-                    await asyncio.sleep(self.async_wait_s)
+                    if rows_counter.value % 25 == 0:
```

Comment:
Perhaps lets make 25 as a static final variable. RESULTS_PAGE_SIZE or sth?

Reply:
Agree, but we need different name. It's not page size. `RESULT_DISPLAY_SLEEP_STEP`?

Reply:
DISPLAY_RESULTS_BATCH_SIZE :smile: ?

Reply:
changed to display_results_batch_size